src/index.js

```diff
@@ -3,7 +3,7 @@ var http = require("http");
 //create a server object:
 http
   .createServer(function(req, res) {
-    res.write("Yoo!"); //write a response to the client
+    res.write("Hey!"); //write a response to the client
```

Comment:
Would this be more fun?

```suggestion
    res.write("Yippee!"); //write a response to the client
```

Reply:
@beepboop-arch what do you think?

Reply:
Yes, go for it!